CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.45.3 / 2024-01-24
+
+This release contains security fixes in dependencies and has been built with go1.21.6. #13450.
+
+* [BUGFIX] TSDB: Remove double memory snapshot on shutdown. #13450
```

Comment:
Shouldn't this be a different PR number than above (which is this PR itself)?

Reply:
ack